includes/User.php

```diff
@@ -1379,7 +1349,7 @@ private function getBlockedStatus( $bFromSlave = true, $shouldLogBlockInStats =
 	 */
 	public function isDnsBlacklisted( $ip, $checkWhitelist = false ) {
 		global $wgEnableSorbs, $wgEnableDnsBlacklist,
-			$wgSorbsUrl, $wgDnsBlacklistUrls, $wgProxyWhitelist;
+			   $wgSorbsUrl, $wgDnsBlacklistUrls, $wgProxyWhitelist;
 
 		if ( !$wgEnableDnsBlacklist && !$wgEnableSorbs )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)